octobot_trading/data/portfolio_profitability.py

```diff
@@ -62,13 +62,13 @@ def __init__(self, config, trader, portfolio_manager, exchange_manager):
 
     async def handle_ticker_update(self, symbol, ticker):
         force_recompute_origin_portfolio = False
-        try:
+        if symbol not in self.origin_crypto_currencies_values:
             # will fail if symbol doesn't have a price in self.origin_crypto_currencies_values and therefore
             # requires the origin portfolio value to be recomputed using this price info in case this price is relevant
-            self.origin_crypto_currencies_values[symbol]
-        except KeyError:
```

Comment:
Why not keeping the try except ?

Reply:
https://github.com/Drakkar-Software/OctoBot-Benchmarks/pull/15